appinterface/projection/crossfire/crossfire.go

```diff
@@ -314,15 +321,15 @@ func (projection *Crossfire) projectCrossfireValidatorView(
 			networkUpgradeBlockheight := msgSubmitSoftwareUpgradeProposalEvent.MsgSubmitSoftwareUpgradeProposalParams.Content.Plan.Height
 
 			// Update Network upgrade target Timestamp in DB
-			network_upgrade_timestamp_dbkey := constants.NETWORK_UPGRADE + constants.DB_KEY_SEPARATOR + "timestamp"
-			errTSUpdate := crossfireChainStatsView.Set(network_upgrade_timestamp_dbkey, networkUpgradeTimestamp.UnixNano())
+			networkUpgradeTimestampDBKey := fmt.Sprintf("%s%s%s", constants.NETWORK_UPGRADE, constants.DB_KEY_SEPARATOR, "timestamp")
+			errTSUpdate := crossfireChainStatsView.Set(networkUpgradeTimestampDBKey, networkUpgradeTimestamp.UnixNano())
 			if errTSUpdate != nil {
 				return fmt.Errorf("error updating network_upgrade timestamp: %v", errTSUpdate)
 			}
 
 			// Update Network upgrade target Blockheight in DB
-			network_upgrade_blockheight_dbkey := constants.NETWORK_UPGRADE + constants.DB_KEY_SEPARATOR + "blockheight"
-			errBlockheightUpdate := crossfireChainStatsView.Set(network_upgrade_blockheight_dbkey, networkUpgradeBlockheight)
+			networkUpgradeBlockheightDBKey := fmt.Sprintf("%s%s%s", constants.NETWORK_UPGRADE, constants.DB_KEY_SEPARATOR, "blockheight")
```

Comment:
How about extract these two DB key out to the constant.go file, since the same key are also used previously.